src/commands/downloadFile.ts

```diff
@@ -61,13 +69,30 @@ export async function download(context: IActionContext, treeItem: IDownloadableT
     ext.outputChannel.appendLog(localize('successfullyDownloaded', 'Successfully downloaded to "{0}".', destinationFolder));
 }
 
-async function getAzCopyDownloads(context: IActionContext, destinationFolder: string, treeItems: IDownloadableTreeItem[]): Promise<IAzCopyDownload[]> {
+export async function downloadSasUrl(context: ISasDownloadContext): Promise<void> {
+    const wizard: AzureWizard<ISasDownloadContext> = new AzureWizard(context, {
+        promptSteps: [new SasUrlPromptStep()],
+        executeSteps: []
+    });
+
+    await wizard.prompt();
+    return await download(context, nonNullProp(context, 'sasUrl'));
+}
+export async function downloadTreeItems(context: IActionContext, treeItem: IDownloadableTreeItem, treeItems?: IDownloadableTreeItem[]): Promise<void> {
+    treeItems = treeItems || [treeItem];
```

Comment:
I think we can be fancier here 🍸 

```suggestion
    treeItems ??= [treeItem];
```

Reply:
Or maybe `||=`, not sure which is better here...